### DISCUSSION TITLE
Copilot suggesting xes instead of doc-comments (Lua)

### DISCUSSION THREAD

**Discussion by fatboychummy** [community/community#106888] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/39929480/1a08ca09-7a0a-47c0-88ef-5757cb68b220)

I am working in Lua and writing doc-comments. Co-pilot used to work fine with this, but recently it has stopped suggesting things like `---@param bla bla`, and is now instead suggesting `xxx@xxxxx bla bla`.

Is there anything that I might have done to cause this, or is this an issue with Copilot itself?

**Comment by SlowGen** [comment]
I'm seeing this too while trying to write GH Actions workflows, any public workflows all come back as xxxxxxx@xx
